Intersect.Editor/Forms/Editors/frmCrafts.cs

```diff
@@ -549,6 +554,10 @@ private void txtSearch_Click(object sender, EventArgs e)
 
         #endregion
 
+        private void darkComboBox1_SelectedIndexChanged(object sender, EventArgs e)
```

Comment:
Rename this function to cmbEvent_SelectedIndexChanged and move to line 454 above
```
#region "Item List - Folders, Searching, Sorting, Etc"
```

Reply:
And then make sure it still compiles/runs and it should be good to go.

Reply:
Done